includes/functions.php

```diff
@@ -2052,7 +2058,10 @@ function device_is_up($device, $record_perf = false)
     $response              = array();
     $response['ping_time'] = $ping_response['last_ping_timetaken'];
     if ($ping_response['result']) {
-        if (isSNMPable($device)) {
+        if ($device['snmp_disable']) {
```

Comment:
It's a good idea to add a check that makes isSNMPable() return true if SNMP is disabled, but it would make the name of the function a bit misleading. Is that fine or should we rename it?

Reply:
Ignore that, but perhaps merge these two code blocks:
```
        if ($device['snmp_disable'] || isSNMPable($device)) {
            $response['status']        = '1';
            $response['status_reason'] = '';
```